docs/source/api/deliveryservice_server_capabilities.rst

```diff
@@ -0,0 +1,222 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-deliveryservice-server-capabilities:
+
+***************************************
+``deliveryservice_server_capabilities``
+***************************************
+
+.. versionadded:: 1.4
+
+``GET``
+=======
+Gets all associations of Server Capabilities to Delivery Services
```

Comment:
"Delivery Services" is a `term`, and needs to be linked to the glossary at the very least the first time it appears within a major section ("page" in an HTML context). Though, ideally, it'd be linked everywhere.